app/queries/servers/post.ts

```diff
@@ -75,6 +75,14 @@ export const queryPostsInThread = (database: Database, rootId: string, sorted =
     return database.get<PostsInThreadModel>(POSTS_IN_THREAD).query(...clauses);
 };
 
+export const queryPostRepliesCount = (database: Database, rootId: string, excludeDeleted = true) => {
```

Comment:
This query queries the post replies. You observe/fetch the count when you use the query. Therefore, I would rename it to `queryPostReplies`.